src/main/java/ru/skypro/flea/converter/LocalDateTimeConverter.java

```diff
@@ -0,0 +1,17 @@
+package ru.skypro.flea.converter;
+
+import javax.persistence.AttributeConverter;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+public class LocalDateTimeConverter implements AttributeConverter<LocalDateTime, Timestamp> {
+  @Override
```

Comment:
i must admit i've rushed about a little too much, sorry)

after my research i've found out that there is no problem when we use **LocalDateTime** in Java and **TIMESTAMP** in Postgres
we do not need to write converters, any annotations in entities are redundant too

i guess there is one annotation that allow us to specify creation time automatically, i'll write about it further

but what about conversion – we shouldn't do anything about it, it works correctly without our intervention
obviously it's my fault and i'm really sorry about it :(